Delicious.html

```diff
@@ -0,0 +1,75 @@
+<!DOCTYPE html>
+<html lang="em">
+<head>
+    <meta charset="utf-8">
+    <title>Delicious</title>
+</head>
+<body>
+	<header>
+		<span><img src="logo.png" alt="logo"></span>
+		<h1>Delicious</h1>
+		<span>The BEST FOOD BLOG ON THE WEB</span>
```

Comment:
This is probably better as a subheading. Something like an h2 has better semantic value.